### ISSUE TITLE
GoogleProvider is not returning results

### ISSUE THREAD

**Issue by johan--** [smeijer/leaflet-geosearch#170]
The GoogleProvider is not returning any results in our application (with a proper Google API key), nor is the demo application working when Google is selected.

**Comment by smeijer** [comment]
The google provider is indeed using the GeoCoding api. Would it be better to use Places? Or do we need a hybrid approach? Places for the suggestions, and the GeoCoding for the selection?

**Comment by itaymining** [comment]
> Hello leafleat-geoserach team Could you please implement places api for google provider? As parameter or as separate provider - both options are good. Places api is better for search, because it can find more locations. Places and geocoding differences: https://cloud.google.com/blog/products/maps-platform/address-geocoding-in-google-maps-apis

you are right. @smeijer please use google "places" autocomplete for autocomplete then grab place_id and get the data from google "geocoder" api

**Comment by smeijer** [comment]
I'd be happy to merge a pull request adding such a provider.

**Comment by jmangel** [comment]
I would love to see this added. I may take a look into the code at some point
